Podfile

```diff
@@ -20,6 +20,8 @@ target 'Multisig' do
   pod 'idn2Swift', :git => 'https://github.com/gnosis/pod-idn2.git', :branch => 'master', :testspecs => ['Tests']
   pod 'BigInt', '4.0.0'
   
+  pod 'Kingfisher/SwiftUI'
```

Comment:
please add a specific version requirement to this line, as an example - see line 21 above

Reply:
done